### DISCUSSION TITLE
[Feature] Show the number of Images for each person directly

### DISCUSSION THREAD

**Discussion by kingvuke** [immich-app/immich#5926] [Feature Request]
### The feature

I would find it very useful if you could directly see the number of pictures per person. At the moment I have to open the person to see the number.

![grafik](https://github.com/immich-app/immich/assets/104008018/6e79ac63-ae3b-42ce-970f-7a563d004710)

### Platform

- [x] Server
- [X] Web
- [ ] Mobile

**Comment by didekoning** [comment]
Made a PR for web: https://github.com/immich-app/immich/pull/25963

**Reply by kingvuke** [reply]
Thank you very much
